internal/scanner/http_client.go

```diff
@@ -81,18 +83,19 @@ func NewHTTPClient(cfg *config.Config) (*HTTPClient, error) {
 	}
 
 	return &HTTPClient{
-		client:        client,
-		headers:       configuredHeaders,
-		hostHeader:    configuredHeaders["Host"],
-		followCookies: cfg.FollowCookies,
-		renewSession:  cfg.RenewSession,
+		client:             client,
+		headers:            configuredHeaders,
+		hostHeader:         configuredHeaders["Host"],
+		followCookies:      cfg.FollowCookies,
+		renewSession:       cfg.RenewSession,
+		isGraphQlAvailable: true,
 	}, nil
 }
 
 func (c *HTTPClient) SendPayload(
 	ctx context.Context,
 	targetURL, placeholderName, encoderName, payload string,
-	testHeaderValue string,
+	degubHeaderValue string,
```

Comment:
What is `degub`? Is it on purpose or is it a typo? `debug`? `debugHeaderValue`?

Reply:
Fixed - https://github.com/wallarm/gotestwaf/pull/167/commits/8a90e824a59e77cb4781533c2d6b78cf34543a5a